src/cryptoadvance/specter/translations/hi/LC_MESSAGES/messages.po

```diff
@@ -490,8 +500,9 @@ msgid "Failed to cancel transaction with RBF. Error: {}"
 msgstr ""
 
 #: src/cryptoadvance/specter/server_endpoints/wallets.py:600
+#, fuzzy
 msgid "Failed to sign PSBT: {}"
-msgstr ""
+msgstr "नोड का कनेक्शन फेल हो गया"
```

Comment:
This translation has omitted the `{}` symbols from the original. If you refer back to the instructions, we need to make sure those brackets are preserved.

Reply:
The brackets are missing in a few other passages as well.